web/components/rsvp/index.js

```diff
@@ -0,0 +1,196 @@
+    const { params } = match;
+    const { id } = params;
+    const {
+      firstName,
+      emergencyContactName,
+      emergencyContactNumber,
+      tShirtSize,
+      dietRestriction,
+      wouldLikeToSee,
+    } = this.state;
+
+    return (
+      <div className="pad-nav application fill-width flex jc-center">
+        <div className="pad-top-mega pad-bottom-tera">
+          <h2>Hi {firstName}, please RSVP <img className="emoji" alt="🙌" src={praise} /></h2>
+          <p>Congrats, we choose you! <img className="emoji" alt="🎉" src={celebrate} /> Please complete this form by 11:59PM on
+            January 5th, 2019 so we can get straight to scheduling you for nwHacks 2019.
+            If we don’t hear from you by the deadline, your spot will be given to
+            someone else and you will be moved to our waitlist.
+          </p>
+          <TextInput
+            label="What is the name of your emergency contact?"
+            name="emergencyContactName"
+            className="margin-top-mega margin-bottom-xxl margin-ends-giga"
+            placeholder="Enter emergency contact name"
+            onChange={newEmergencyContactName => this.setState({
+              emergencyContactName: newEmergencyContactName })}
+            // value={emergencyContactName}
+            // onBlur={() => this.setFieldAsBlurred('githubLink')}
+            // error={this.getErrorIfBlurred('githubLink')}
+          />
+          <TextInput
+            label="What is the number of your emergency contact?"
+            name="emergencyContactNumber"
+            className="margin-top-mega margin-bottom-xxl margin-ends-giga"
+            placeholder="123-456-7890"
+            onChange={newEmergencyContactNumber => this.setState({
+              emergencyContactNumber: newEmergencyContactNumber })}
+            // value={emergencyContactNumber}
+            // onBlur={() => this.setFieldAsBlurred('githubLink')}
+            // error={this.getErrorIfBlurred('githubLink')}
+          />
+          <Select
+            options={tShirtSizes}
+            name="t-shirt-size"
+            label="What is your t-shirt size? (note: these are unisex clothing sizes)"
+            className="margin-top-mega margin-bottom-xxl"
+            onChange={({ value: newTShirtSize }) => console.log(newTShirtSize)}
+            // value={newTShirtSize}
+            // onBlur={() => this.setFieldAsBlurred('source')}
+            // error={this.getErrorIfBlurred('source')}
+          />
+          <Select
+            options={dietRestrictions}
+            name="dietRestrictions"
+            label="Do you have any dietary restrictions or allergies?"
+            placeholder="Enter an option or choose from the dropdown"
+            className="margin-top-mega margin-bottom-xxl"
+            onChange={({ value: newDietRestriction }) => this.setState({
+              dietRestriction: newDietRestriction })}
+            // value={dietRestriction}
```

Comment:
`value` controls the current value. Your select was probably fine before, it likely broke because I edited the Select component in my last commit to make value of `undefined` mean "no value" (previously undefined counted as a value which wasn't the expected behaviour).

https://github.com/nwplus/nwhacks2019/blob/a05bfa284bf444c8fe4023bb858dbfe9e43ca64f/web/components/input/select/index.js#L83-L84